great-idea-website/index.html

```diff
@@ -15,49 +15,100 @@
+   
+    <!--Features Container-->
+    <div class="top_content">
+      <h2>Features</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    
+    <!--About Container-->
+    <div class="top_content">
+      <h2>About</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    
+    <!--Image Container-->
+    <div class="content_image">
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    </div>
+    
+    <!--Services Container-->
+    <div class="bottom_content">
+      <h2>Services</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    
+    <!--Product Container-->
+    <div class="bottom_content">
+      <h2>Product</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    
+    <!--Vision Container-->
+    <div class="bottom_content">
+      <h2>Vision</h2>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    
+  </div>
   
-  Services
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  Product
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
+  <!--Page Break-->
+  <hr >
+  
+  <!--Contact section-->
+  <div id="contact_container">
+    
+      <!--Contact Container-->
+    <div class="bottom_content">
+      <h2>Contact</h2>
+      <adress>123 Way 456 Street<br>
+          Somewhere, USA<br>
+          1 (888) 888-8888<br>
+          sales@greatidea.io
+      </adress>
+    </div>  
+  </div>
+  
+  <!--Footer section-->
```

Comment:
In this situation, we could have used a `footer` element to group the contact section and the copyright together in a way that provides more meaning to the document, rather than just having the copyright in the footer by itself.